workflow/rules/common.smk

```diff
@@ -112,6 +112,29 @@ primer_panels = (
 )
 
 
+def is_activated(xpath):
+    c = config
+    for entry in xpath.split("/"):
+        c = c.get(entry, {})
+    return bool(c.get("activate", False))
+
+
+custom_alignment_props = (
+    (
+        pd.read_csv(
+            config["custom_alignment_properties"]["tsv"],
```

Comment:
Doesn't this fail if there is no global tsv?

Reply:
Yes, but I would expect the user to define one in case this option has been activated.